src/client/renderer/r_mesh_model.c

```diff
@@ -903,8 +904,8 @@ static void R_LoadObjShellVertexArrays(r_model_t *mod, r_obj_t *obj, GLuint *ele
 
 			unique_vertex_list = g_array_append_vals(unique_vertex_list, &(const r_obj_shell_interleave_vertex_t) {
 				.vertex = { ve->point[0], ve->point[1], ve->point[2] },
-				.normal = { ve->normal[0], ve->normal[1], ve->normal[2] },
-				.diffuse = { (u16vec_t) (ve->texcoords[0] * USHRT_MAX), (u16vec_t) (ve->texcoords[1] * USHRT_MAX) }
+				 .normal = { ve->normal[0], ve->normal[1], ve->normal[2] },
+				  .diffuse = { (u16vec_t) (ve->texcoords[0] * USHRT_MAX), (u16vec_t) (ve->texcoords[1] * USHRT_MAX) }
```

Comment:
Yeah I had a function for it initially but I removed it for some reason. I'll re-introduce it.